src/main/java/core/basesyntax/model/User.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class User {
+    private long id;
```

Comment:
we will have id's in all of our entities in the future, but for now, let's use other values
```suggestion
    private int age;
```